coresimd/arm/armclang.rs

```diff
@@ -0,0 +1,30 @@
+//! ARM compiler specific intrinsics
+//!
+//! # References
+//!
+//! - [ARM Compiler v 6.10 - armclang Reference Guide](https://developer.arm.com/docs/100067/0610)
+
+/// This intrinsic inserts a BKPT instruction into the instruction stream generated by the compiler
+///
+/// It enables you to include a breakpoint instruction in your Rust code
+///
+/// `val` is a compile-time constant integer whose range is:
+///
+/// - `0...65535` if you are compiling source as A32 code.
```

Comment:
This should also be removed since it currently won't work and can be added later once it works in a backwards compatible way.

Reply:
Ah sorry, I see you also updated the Note below. So I think this is fine. I'll merge once CI is green. Let me know if there are any spurious CI failures.